src/app/widgets/Visualizer/index.jsx

```diff
@@ -744,6 +744,7 @@ class VisualizerWidget extends PureComponent {
             port: controller.port,
             units: store.get('workspace.units', METRIC_UNITS),
             theme: this.config.get('theme'),
+            showSoftLimitsWarning: this.config.get('showSoftLimitsWarning') ? this.config.get('showSoftLimitsWarning') : false,
```

Comment:
WidgetConfig.get accepts a default value as a secondary argument, no need for ternary operation